### PR TITLE
Init first support of the magic word

### PR DESCRIPTION
**Pull request by MalloZup**
- todo:

- [x] Remove the comment for triggering the job, once gitbot is done. (otherwise inf loop)

### CODE REVIEW

**Comment by MalloZup** [comment]
@gitbot nice work, redo the test

File: gitbot.rb [hunk 1]
```diff
@@ -43,6 +43,18 @@ def create_comment(repo, pr, comment)
   @client.create_commit_comment(repo, pr, comment)
 end
 
+# this function will check if the PR contains in comment the magic word
```
Comment (MalloZup) [inline gitbot.rb]: i am a review comment

**Comment by MalloZup** [comment]
@gitbot rerun the macarena

**Comment by MalloZup** [comment]
- todo:

-  since we cannot retrigger all the test, we need to do:
1. "@gitbot rerun CONTEXT (e.g. pysthon-test), macarena "
2. this will trigger only the specific test, and remove the comment were gitbot is triggered (otherwise there is an infinite loop)

**Comment by MalloZup** [comment]
HEY @gitbot rerun gitbot-magic please

**Comment by MalloZup** [comment]
Ciao @gitbot rerun gitbot-magic !!!

**Comment by MalloZup** [comment]
Ciao @gitbot rerun gitbot-magic !!!!!!! (working)

**Comment by MalloZup** [comment]
2nd comment

**Comment by MalloZup** [comment]
@gitbot rerun gitbot-magic !!!

**Comment by MalloZup** [comment]
- how it works:


basically we can retrigger specific tests, i didn"t add explicitly a mechanism to retrigger all tests because this was not stable and safe enough.

when for example we say in a comment:

@gitbot rerun Test-name !!! , this will trigger the tests.

As a consequence the comment will be erased, is kind of locking, because otherwise we have infinite loops.

**Comment by MalloZup** [comment]
### comment:
```console
@gitbot rerun gitbot-magic !!! , this will trigger the tests.
```

**Comment by mc** [comment]
wat, i dont even...?

**Comment by MalloZup** [comment]
@mc sorry wrong nick :)

**Comment by MalloZup** [comment]
@mcalmer  i will merge this today

**Pull request merged**